### PR TITLE
Doc: support fetch n from subscription cursor

### LINKED ISSUE #2286: Doc: support fetch n from subscription cursor
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/16764

### Which part(s) of the docs might be affected or should be updated? And how?

Able to fetch multiple rows from subscriptions cursor using `FETCH n FROM cursor` where n is an integer.

### Reference

_No response_

### PR DESCRIPTION
**Pull request by WanYixian**
## Info

- **Description**

  - Support fetch n from subscription cursor

- **Related code PR**

  - https://github.com/risingwavelabs/risingwave/pull/16764

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/2286

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/transform/subscription.md [hunk 1]
```diff
@@ -70,6 +72,14 @@ Note that each time `FETCH NEXT FROM cursor_name` is called, it will return one
...
 This method is non-blocking. Even if the current table has no new incremental data, `FETCH NEXT FROM cursor_name` will not block, but will return an empty row. When new incremental data is generated, calling this statement again will return the latest row of data.
 
+#### `FETCH n FROM cursor_name`
+
+You also can fetch multiple rows at once from the cursor using the FETCH n FROM cursor_name command. `n` is the number of rows to fetch.
+
+```sql
+FETCH n FROM cursor_name;
+```
+
```
Comment (WanYixian) [inline docs/transform/subscription.md:82]: @xxhZs  Let me know if any notes are required, thanks!

File: docs/transform/subscription.md [hunk 2]
```diff
@@ -70,6 +72,14 @@ Note that each time `FETCH NEXT FROM cursor_name` is called, it will return one
 
 This method is non-blocking. Even if the current table has no new incremental data, `FETCH NEXT FROM cursor_name` will not block, but will return an empty row. When new incremental data is generated, calling this statement again will return the latest row of data.
 
+#### `FETCH n FROM cursor_name`
+
+You also can fetch multiple rows at once from the cursor using the `FETCH n FROM cursor_name` command. `n` is the number of rows to fetch.
+
+```sql
+FETCH n FROM cursor_name;
```
Comment (ShanlanLi) [inline docs/transform/subscription.md:80]: Could you provide an example here to show the output?

**Review by ShanlanLi — APPROVED** [review]
LGTM

File: docs/transform/subscription.md (hunk 2 above)
Comment (ShanlanLi) [reply docs/transform/subscription.md:80]: FYI, the code in the example section can be used to create a subscription and then use fetch n from cursor.

Comment (xxhZs) [reply docs/transform/subscription.md:80]: An example may not be necessary, just show that he called fetch next the equivalent of n times.

**Review by xxhZs — APPROVED** [review]
lgtm

**Pull request merged**